internal/config/config.go

```diff
@@ -84,7 +84,7 @@ type MTLS struct {
 func LoadConfig(filePath string, fileName string) (*Config, error) {
 	v := viper.New()
 
-	// We need to assure that defaults have been set in order to bind env variables.
+	// We need to assure that defaults have been set in order to bind env variables
```

Comment:
Bring the trailing dot back 😠

Reply:
https://github.com/newrelic/newrelic-infra-operator/blob/ce13dfff70008c54ed0598bb6904b0cf8fed9808/.golangci.yml#L66

https://github.com/tetafro/godot

Ok. Omw.

Reply:
I though the same, but it is a multiline comment